src/Resources/config/routing/admin/group.yml

```diff
@@ -0,0 +1,9 @@
+bitbag_sylius_product_attribute_groups_plugin:
+  resource: |
+    alias: bitbag_sylius_product_attribute_group_plugin.group
+    except: ['show']
+    section: admin
+    templates: "@SyliusAdmin/Crud"
+    grid: bitbag_sylius_product_attribute_group
+  type: sylius.resource
+  prefix: /admin/product-attribute-groups-plugin
```

Comment:
You added `admin` prefix while importing this file in a `routing.yaml`. Remove `prefix`, add `product-attribute-groups-plugin` as path under `resource` configuration please. Here's a file you can reference to https://github.com/BitBagCommerce/SyliusMailTemplatePlugin/blob/master/src/Resources/config/routing/admin/email_template.yaml.